packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -53,6 +53,7 @@ const enableKernelGC = true;
 // v$NN.t.endPosition = $NN
 // v$NN.vs.$key = string
 // v$NN.lastSnapshot = JSON({ snapshotID, startPos })
+// snapshot.$id = [vatID, ...]
```

Comment:
let's add a note, `// exclude from consensus`, to both this and the `v$NN.lastSnapshot` keys: when #3442 lands we need to not hash these DB keys into the consensus state